src/meta/.htaccess

```diff
@@ -0,0 +1,984 @@
+# Apache Server Configs v2.14.0 | MIT License
```

Comment:
I don't think this should be our default.
There's many things set up here that make no sense.
One example: the error page setup when we don't even have one